tests/data/yaml_include_txt.md

```diff
@@ -0,0 +1,6 @@
+# header1
```

Comment:
What is this?

Reply:
Just a test file to verify that `!include` directive will not only inject yamls into yaml, but also plain text files (as simple string)